src/app.js

```diff
@@ -1,21 +1,42 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const path = require('path');
+const fs = require('fs');
+const myConsole = require('console');
+
+function main() {
+  if (process.argv.length !== 4) {
+    myConsole.log('Error! You should start program with'
+      + 'file that you want to move, and location to paste it');
+
+    return;
+  }
+
+  const [,, file, insertTo] = process.argv;
+
+  const fileLocation = path.join(__dirname, file);
+  let insertLocation = path.join(__dirname, insertTo);
+
+  if (fileLocation === insertLocation) {
+    myConsole.log('Error! Same name and location');
```

Comment:
You can use console.error here, as it outputs to STDERR.
```suggestion
    myConsole.error('Error! Same name and location');
```